lib/storage/src/content_manager/consensus/persistent.rs

```diff
@@ -18,6 +18,7 @@ use crate::types::PeerAddressById;
 use crate::StorageError;
 
 const STATE_FILE_NAME: &str = "raft_state";
+const STATE_FILE_NAME_JSON: &str = "raft_state.json";
```

Comment:
I'd just suggest to change this for code clarity

```suggestion
const STATE_FILE_NAME_CBOR: &str = "raft_state";
const STATE_FILE_NAME: &str = "raft_state.json";
```
or 
```suggestion
const STATE_FILE_NAME_CBOR: &str = "raft_state";
const STATE_FILE_NAME_JSON: &str = "raft_state.json";
```

Reply:
done